### PR TITLE
Lambda configuration tweaks

### PR DESCRIPTION
**Pull request by firstmorecoffee**
Use a timeout when connecting to SSM so we notice VPC issues.

Build in amd64 explicitly for people building on M1 macs.

### CODE REVIEW

File: advanced/mysql_temp_user_strategy/lambda_src/config.py [hunk 1]
```diff
@@ -46,7 +48,12 @@ def _get_db_password(session: Session) -> str:
 
     db_password_key = os.environ.get("DB_PASSWORD_KEY", "/symops.com/DB_PASSWORD")
 
-    ssm = session.client("ssm")
+    # Use an aggressive timeout here so we get a helpful message during initialization
+    # if the VPC cannot reach SSM. You can max this more lax if necessary.
```
Comment (Arianna2028) [inline advanced/mysql_temp_user_strategy/lambda_src/config.py:52]:
```suggestion
    # if the VPC cannot reach SSM. You can make this more lax if necessary.
```

File: advanced/postgres_role_strategy/lambda_src/config.py [hunk 2]
```diff
@@ -34,6 +37,11 @@ def get_pg_password() -> str:
 
     pg_password_key = os.environ.get("PG_PASSWORD_KEY", "/symops.com/PG_PASSWORD")
 
-    ssm = boto3.client("ssm")
+    # Use an aggressive timeout here so we get a helpful message during initialization
+    # if the VPC cannot reach SSM. You can max this more lax if necessary.
```
Comment (Arianna2028) [inline advanced/postgres_role_strategy/lambda_src/config.py:41]:
```suggestion
    # if the VPC cannot reach SSM. You can make this more lax if necessary.
```

File: advanced/postgres_temp_user_strategy/lambda_src/config.py [hunk 3]
```diff
@@ -46,7 +50,12 @@ def _get_db_password(session: Session) -> str:
 
     db_password_key = os.environ.get("DB_PASSWORD_KEY", "/symops.com/DB_PASSWORD")
 
-    ssm = session.client("ssm")
+    # Use an aggressive timeout here so we get a helpful message during initialization
+    # if the VPC cannot reach SSM. You can max this more lax if necessary.
```
Comment (Arianna2028) [inline advanced/postgres_temp_user_strategy/lambda_src/config.py:54]:
```suggestion
    # if the VPC cannot reach SSM. You can make this more lax if necessary.
```

File: advanced/mysql_temp_user_strategy/lambda_src/config.py (hunk 1 above)
Comment (llam15) [inline advanced/mysql_temp_user_strategy/lambda_src/config.py:52]:
```suggestion
    # if the VPC cannot reach SSM. You can make this more lax if necessary.
```

File: advanced/postgres_role_strategy/lambda_src/config.py (hunk 2 above)
Comment (llam15) [inline advanced/postgres_role_strategy/lambda_src/config.py:41]:
```suggestion
    # if the VPC cannot reach SSM. You can make this more lax if necessary.
```

**Review by Arianna2028 — APPROVED** [review]
Is the amd64 thing just a default for non-M1 or might this cause issues for people using non-M1 macs?

File: advanced/postgres_temp_user_strategy/lambda_src/config.py (hunk 3 above)
Comment (llam15) [inline advanced/postgres_temp_user_strategy/lambda_src/config.py:54]:
```suggestion
    # if the VPC cannot reach SSM. You can make this more lax if necessary.
```

**Review by llam15 — APPROVED** [review]

**Pull request merged**